proto/target.proto

```diff
@@ -249,17 +246,51 @@ message TargetStats {
   int64 failed_runs = 6;
 }
 
+message AggregateTargetStats {
+  // The target label that these stats are for.
+  string label = 1;
+
+  TargetStatsData data = 2;
+}
+
 message GetTargetStatsResponse {
   context.ResponseContext response_context = 1;
   // Per-target stats on observed flakes.  Currently, flakes are computed based
   // on CI runs on master--it's difficult to assert "sequentialness" of non-CI
   // runs, and moreover, individual users can cause random weird-looking
   // failures locally all the time.
-  repeated TargetStats stats = 2;
+  repeated AggregateTargetStats stats = 2;
+}
 
-  // If the user didn't specify a set of targets to fetch, there may be multiple
-  // pages of results.
-  string next_page_token = 3;
+message GetDailyTargetStatsRequest {
+  context.RequestContext request_context = 1;
+
+  // If specified, a list of targets for which we should fetch flake data.
+  // Daily stats will be aggregated over this set of targets--if no targets are
+  // specified, total stats for all
+  repeated string labels = 2;
+
+  // If specified, stats will be restricted to invocations in this repo.
+  string repo = 3;
+
+  // If many/all labels are requested, there may be multiple pages of results.
+  string page_token = 4;
+}
+
+message DailyTargetStats {
+  // YYYY-MM-DD
+  string date = 1;
```

Comment:
maybe add a TODO to use a Real Date Type™

Reply:
filed https://github.com/buildbuddy-io/buildbuddy-internal/issues/3533 since we have this in a few places.